btr-api/tests/postman/btr-api-DEV.postman_environment.json

```diff
@@ -0,0 +1,81 @@
+{
+	"id": "1affd88f-d643-46d3-a4de-27877683af24",
+	"name": "btr-api-DEV",
+	"values": [
+		{
+			"key": "base_url",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "internal_url",
+			"value": "https://btr-api-dev-nr4i2ahhhq-nn.a.run.app",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "version",
+			"value": "api/v1",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "token",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "realm",
+			"value": "bcregistry",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "auth_api_url",
+			"value": "https://auth-api-dev.apps.silver.devops.gov.bc.ca",
```

Comment:
this can go through the gateway

Reply:
oh nice okay I'll update

Reply:
Oh no wait then I'll need an api key etc -- I think for now I will just leave this